superwise/assets/dashboards/superwise.json

```diff
@@ -0,0 +1 @@
eseries","queries":[{"search":{"query":""},"data_source":"incident_analytics","compute":{"aggregation":"count"},"name":"query1","indexes":["*"],"group_by":[]}],"style":{"palette":"purple","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"include_zero":true,"scale":"linear","label":"","min":"auto","max":"auto"},"markers":[]},"layout":{"x":0,"y":3,"width":4,"height":3}}]},"layout":{"x":4,"y":0,"width":4,"height":7}},{"id":5132323858991144,"definition":{"title":"Metrics","type":"group","background_color":"vivid_purple","show_title":true,"layout_type":"ordered","widgets":[{"id":2728068484701532,"definition":{"title":"Total # of Predictions","title_size":"16","title_align":"left","type":"query_value","requests":[{"formulas":[{"formula":"query1"}],"response_format":"scalar","queries":[{"query":"sum:superwise.metric.overall.quantity{*}","data_source":"metrics","name":"query1","aggregator":"sum"}]}],"autoscale":true,"precision":0},"layout":{"x":0,"y":0,"width":2,"height":3}},{"id":2448869580205054,"definition":{"title":"Active Models","title_size":"16","title_align":"left","type":"query_value","requests":[{"formulas":[{"formula":"count_not_null(query1)"}],"conditional_formats":[{"comparator":">","palette":"white_on_green","value":1}],"response_format":"scalar","queries":[{"query":"sum:superwise.metric.overall.data_drift{*} by {model_name}","data_source":"metrics","name":"query1","aggregator":"avg"}]}],"autoscale":true,"precision":2},"layout":{"x":2,"y":0,"width":2,"height":3}},{"id":7366254108548012,"definition":{"title":"Models Input Drift From Baseline","title_size":"16","title_align":"left","type":"toplist","requests":[{"formulas":[{"formula":"query1","limit":{"count":5,"order":"desc"}}],"conditional_formats":[{"comparator":"<","palette":"white_on_green","value":50},{"comparator":">=","palette":"white_on_red","value":80},{"comparator":">=","palette":"white_on_yellow","value":50}],"response_format":"scalar","queries":[{"query":"max:superwise.metric.overall.data_drift{*} by {model_name}","data_source":"metrics","name":"query1","aggregator":"last"}]}]},"layout":{"x":0,"y":3,"width":4,"height":3}},{"id":1954347328001564,"definition":{"title":"Total # of Predictions - Trend","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"horizontal","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"formula":"query1"}],"response_format":"timeseries","queries":[{"query":"max:superwise.metric.overall.quantity{*} by {model_name}","data_source":"metrics","name":"query1"}],"style":{"palette":"warm","line_type":"solid","line_width":"normal"},"display_type":"area"}],"yaxis":{"include_zero":true,"scale":"linear","label":"","min":"auto","max":"auto"},"markers":[]},"layout":{"x":0,"y":6,"width":4,"height":3}}]},"layout":{"x":8,"y":0,"width":4,"height":10}}],"template_variables":[],"layout_type":"ordered","is_read_only":false,"notify_list":[],"reflow_type":"fixed","id":"3rs-pu9-3be"}
```

Comment:
The dashboard is looking great, just had a few minor comments:

- In your text widget, regarding the last few sentences:

    _To read more about how to define which incidents should be sent to Datadog, **read more here**. For further customizations read our integration: **docs**_

    The "_**read more here**_" is missing a link. For the second sentence, changing it to "_For further dashboard customizations, see our integration documentation_" seems more clear.

- For the incidents widget, what is the "# of open incidents in the last time" referring to? Maybe "# of incidents opened in the last x hours" or something along those lines would make more sense (just a suggestion).

Reply:
I fixed both comments (👍🏻

Reply:
Sorry, for the "Last X hours" comment I meant that it would be helpful to provide a number - i.e. "in the last 24 hours"  or "in the last hour" or whatever the proper measurement is for that metric. Also, I'd recommend changing the last sentence in the text widget to:

 For further dashboard customizations, see our **integration documentation**

Thanks!

Reply:
Hi, the time depends on the resolution of the user, so maybe X is a good suggestion.
The last sentence in the widget already contains the sentence "For further dashboard customizations, see our integration documentation"

Reply:
@anawishnoff

Reply:
@oryan-omer  Ah I see - if it's based on the time since the last resolution of the user, changing it to "Number of incidents opened since last resolution" makes more sense to me. Apologies for the back and forth. 

For the last sentence, I was just suggesting to put the hyperlink over the bolded text below, rather than include the hyperlink over "docs":

For further dashboard customizations, see our **integration documentation**

Reply:
gotchya, Fixed (: @anawishnoff